readme.txt

```diff
@@ -93,6 +93,42 @@ We currently accept the following payment methods:
 - MyBank
 - Sofort
 
+== Frequently Asked Questions ==
```

Comment:
I moved this section up in an attempt to make it more visible on the plugin store page. Right now it's stuck at the bottom of the page, so I'm seeing if I can move it up 🎲